src/LeanMapper/DefaultEntityFactory.php

```diff
@@ -19,7 +19,7 @@
 class DefaultEntityFactory implements IEntityFactory
 {
 
-    /*
+    /**
```

Comment:
Aha, to jsem netušil. A pořád to asi vyžaduje, viz dokumentace z [poslední verze](https://codedoc.pub/inlm/leanmapper/v3.1.1/class-LeanMapper.DefaultEntityFactory.html) a z [tohoto PR](https://codedoc.pub/inlm/leanmapper/pr/travis-code-checker/class-LeanMapper.DefaultEntityFactory.html).

Zkusím vymyslet, jak by se to dalo vyřešit.